supply-chain-security/compromises/2024/targeted-signed-endoor.md

```diff
@@ -0,0 +1,25 @@
+# Malware Disguised as Installer from Korean Public Institution
```

Comment:
```suggestion
# Malware Disguised as Official Installer Used to Target Korean Public Institution
```

This will clarify that the malware was used to target the institution, not that it originated from there.